src/os_msg.eliomi

```diff
@@ -37,6 +37,18 @@ val msg :
   ?level:[`Err | `Msg] -> ?duration:float -> ?onload:bool -> string -> unit
 
 [%%server.start]
+
+(** Set to true if an activation key has been already created and sent to the
+ * user email, else false. Default is false.
+ *)
 val activation_key_created : bool Eliom_reference.Volatile.eref
+
+(** ((firstname, lastname), (password, password_confirmation)) option
```

Comment:
Can you put it in `[ ]` so that Ocamldoc will render it as monospace? Same wherever code is needed.